change/@fluentui-react-tags-preview-ec023340-9146-4eb3-b535-b0d3230ecbac.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28884&commitId=29ce896d89344046d774814897861fcb34271d03&env=prod&clientType=FLUENTUI)


  <details><summary><b>InteractionTag Converged</b> 9 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|InteractionTag Converged.Brand.hover dismiss.chromium.png|756| Changed|
|InteractionTag Converged.Brand - High Contrast.hover dismiss.chromium.png|91| Changed|
|InteractionTag Converged.Brand - Dark Mode.hover dismiss.chromium.png|169| Changed|
|InteractionTag Converged.Filled - High Contrast.hover dismiss.chromium.png|2422| Changed|
|InteractionTag Converged.Filled.hover dismiss.chromium.png|471| Changed|
|InteractionTag Converged.Filled - Dark Mode.hover dismiss.chromium.png|41| Changed|
|InteractionTag Converged.Outline - Dark Mode.hover dismiss.chromium.png|100| Changed|
|InteractionTag Converged.Outline.hover dismiss.chromium.png|507| Changed|
|InteractionTag Converged.Outline - High Contrast.hover dismiss.chromium.png|2335| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

Reply:
The new screenshot is correct. I must have accepted wrong screenshot before 🙈